osmaxx/conversion_api/coordinate_reference_systems.py

```diff
@@ -0,0 +1,162 @@
+    ('32710', _('UTM Zone 10, southern hemisphere')),
+
+    ('32711', _('UTM Zone 11, southern hemisphere')),
+    ('32712', _('UTM Zone 12, southern hemisphere')),
+    ('32713', _('UTM Zone 13, southern hemisphere')),
+    ('32714', _('UTM Zone 14, southern hemisphere')),
+    ('32715', _('UTM Zone 15, southern hemisphere')),
+    ('32716', _('UTM Zone 16, southern hemisphere')),
+    ('32717', _('UTM Zone 17, southern hemisphere')),
+    ('32718', _('UTM Zone 18, southern hemisphere')),
+    ('32719', _('UTM Zone 19, southern hemisphere')),
+    ('32720', _('UTM Zone 20, southern hemisphere')),
+
+    ('32721', _('UTM Zone 21, southern hemisphere')),
+    ('32722', _('UTM Zone 22, southern hemisphere')),
+    ('32723', _('UTM Zone 23, southern hemisphere')),
+    ('32724', _('UTM Zone 24, southern hemisphere')),
+    ('32725', _('UTM Zone 25, southern hemisphere')),
+    ('32726', _('UTM Zone 26, southern hemisphere')),
+    ('32727', _('UTM Zone 27, southern hemisphere')),
+    ('32728', _('UTM Zone 28, southern hemisphere')),
+    ('32729', _('UTM Zone 29, southern hemisphere')),
+    ('32730', _('UTM Zone 30, southern hemisphere')),
+
+    ('32731', _('UTM Zone 31, southern hemisphere')),
+    ('32732', _('UTM Zone 32, southern hemisphere')),
+    ('32733', _('UTM Zone 33, southern hemisphere')),
+    ('32734', _('UTM Zone 34, southern hemisphere')),
+    ('32735', _('UTM Zone 35, southern hemisphere')),
+    ('32736', _('UTM Zone 36, southern hemisphere')),
+    ('32737', _('UTM Zone 37, southern hemisphere')),
+    ('32738', _('UTM Zone 38, southern hemisphere')),
+    ('32739', _('UTM Zone 39, southern hemisphere')),
+    ('32740', _('UTM Zone 40, southern hemisphere')),
+
+    ('32741', _('UTM Zone 41, southern hemisphere')),
+    ('32742', _('UTM Zone 42, southern hemisphere')),
+    ('32743', _('UTM Zone 43, southern hemisphere')),
+    ('32744', _('UTM Zone 44, southern hemisphere')),
+    ('32745', _('UTM Zone 45, southern hemisphere')),
+    ('32746', _('UTM Zone 46, southern hemisphere')),
+    ('32747', _('UTM Zone 47, southern hemisphere')),
+    ('32748', _('UTM Zone 48, southern hemisphere')),
+    ('32749', _('UTM Zone 49, southern hemisphere')),
+    ('32750', _('UTM Zone 50, southern hemisphere')),
+
+    ('32751', _('UTM Zone 51, southern hemisphere')),
+    ('32752', _('UTM Zone 52, southern hemisphere')),
+    ('32753', _('UTM Zone 53, southern hemisphere')),
+    ('32754', _('UTM Zone 54, southern hemisphere')),
+    ('32755', _('UTM Zone 55, southern hemisphere')),
+    ('32756', _('UTM Zone 56, southern hemisphere')),
+    ('32757', _('UTM Zone 57, southern hemisphere')),
+    ('32758', _('UTM Zone 58, southern hemisphere')),
+    ('32759', _('UTM Zone 59, southern hemisphere')),
+    ('32760', _('UTM Zone 60, southern hemisphere')),
+)
+
+CRS_CHOICES = (
+    (
+        _('Global coordinate reference systems'),
+        GLOBAL_CRS,
+    ),
+    (
+        _('UTM zones'),
+        UTM_ZONES,
+    ),
+)
```

Comment:
I think in this specific case where the inner tuples are constants rather than literals, I'd prefer the more compact
```python
CRS_CHOICES = (
    (_('Global coordinate reference systems'), GLOBAL_CRS),
    (_('UTM zones'), UTM_ZONES),
)
```

If you want to keep is as is for consistency with all-literal choices, that's fine with me, too, though.